src/pages/regulatory/index.js

```diff
@@ -239,6 +239,10 @@ const Regulatory = () => {
                                                 target="_blank"
                                                 to="https://beta.gamblingcommission.gov.uk/public-register/business/detail/39495"
                                                 rel="noopener noreferrer"
+                                                style={{
+                                                    color: `var(--color-black-3)`,
+                                                    textDecoration: `none`,
```

Comment:
Nit: Can we use `''` instead of backticks

Reply:
done.